### ISSUE TITLE
RCS1097 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#87]
The following code doesn't throw RCS1097 Remove redundant 'ToString' call.

```csharp
string z = new Object().ToString() + "test";
```
